### PR TITLE
D17 58/navbar mobilka

### PR DESCRIPTION
**Pull request by yoinker57**
Added tabbar + routing

### CODE REVIEW

File: app/(tabs)/index.tsx [hunk 1]
```diff
@@ -1,31 +1,32 @@
...
-    </View>
-  );
+export default function Home() {
+    return (
+        <View style={styles.container}>
+            <Text style={styles.title}>Home</Text>
+            <View
+                style={styles.separator}
+                lightColor="#eee"
+                darkColor="rgba(255,255,255,0.1)"
```
Comment (irosikoni) [inline app/(tabs)/index.tsx]: don't mix rgba with hex, decide on one of them

File: app/(tabs)/reservation.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+import { Link } from 'expo-router'
+
+export default function Reservations() {
+    return (
+        <View style={styles.container}>
+            <Text style={styles.title}>Reservations</Text>
+            <View
+                style={styles.separator}
+                lightColor="#eee"
+                darkColor="rgba(255,255,255,0.1)"
```
Comment (irosikoni) [inline app/(tabs)/reservation.tsx]: you are using this colors in more then one place, so maybe put them in Color.ts file

**Review by irosikoni — CHANGES_REQUESTED** [review]
I don't like the fact that navigation bar is disappearing on reservation page. The rest is perfectly fine, only small adjustments needed in code

File: app/(tabs)/index.tsx (hunk 1 above)
Comment (yoinker57) [reply app/(tabs)/index.tsx]: we will remove it when we create the home screen

**Review by irosikoni — CHANGES_REQUESTED** [review]
there is no possibility to go to room page

**Review by irosikoni — COMMENTED** [review]
3f34381e-8ca1-4489-9d49-dc2aba7f726a

Make padding bigger in navigation bar

File: app/_layout.tsx [hunk 3]
```diff
@@ -1,58 +1,91 @@
...
+    )
 }
+
+
+const styles = StyleSheet.create({
+    image: {
+        width: 50,
+        height: 50,
+    },
+})
```
Comment (wleklinskimateusz) [inline app/_layout.tsx]: Screenshot 2024-05-06 at 20 41 03
end of file line missing for some reason

**Review by wleklinskimateusz — APPROVED** [review]
Looks Good to me

**Review by irosikoni — APPROVED** [review]

**Pull request merged**